ui/src/pages/tasks.tsx

```
@@ -0,0 +1,48 @@
+      <h2 class="text-2xl font-semibold mb-4">Your Tasks</h2>
+      <div class="mt-4"></div>
+      <Show when={tasks.isError}>
+        <div>Error: {tasks.error && tasks.error.message}</div>
+      </Show>
+      <Show when={tasks.isLoading}>
+        <Spinner />
+      </Show>
+      <Show when={tasks.isSuccess}>
+        <div>
+          {tasks.data ? (
+            tasks.data.map((task) => <TaskBox taskDetails={task} />)
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)